data/src/main/java/org/apache/iceberg/data/TableMigrationUtil.java

```diff
@@ -215,11 +215,11 @@ private static DataFile buildDataFile(
         .build();
   }
 
-  private static ExecutorService migrationService(int concurrentDeletes) {
+  private static ExecutorService migrationService(int numThreads) {
```

Comment:
Took another look at this procedure, I think we can just call this `parallelism`